File: event-service/src/main/java/org/tggc/eventservice/EventServiceApplication.java
package org.tggc.eventservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.scheduling.annotation.EnableScheduling;
import org.tggc.userapi.api.UserApi;

@SpringBootApplication
@EnableScheduling
@EnableFeignClients(clients = {UserApi.class})
public class EventServiceApplication {

	public static void main(String[] args) {
		SpringApplication.run(EventServiceApplication.class, args);
	}

}

File: event-service/src/main/kotlin/org/tggc/eventservice/EventServiceApplication.kt
package org.tggc.eventservice

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling
import org.tggc.userapi.api.UserApi
import reactivefeign.spring.config.EnableReactiveFeignClients

@SpringBootApplication
@EnableScheduling
@EnableReactiveFeignClients(clients = [UserApi::class])
open class EventServiceApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(EventServiceApplication::class.java, *args)
        }
    }
}
